### PR TITLE
feat(ui): Implement Resources listing

### PR DESCRIPTION
**Pull request by bdauria**
# Pull Request Template

## Description

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [x] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

**Comment by bdauria** [comment]
area/ui

File: webpack.config.dev.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.exports = merge(baseConfig, devConfig, {
     alias: {
       react: path.resolve('./node_modules/react'),
       'react-router-dom': path.resolve('./node_modules/react-router-dom'),
+      '@material-ui/core': path.resolve('./node_modules/@material-ui/core'),
```
Comment (kduret) [inline webpack.config.dev.js]: why do we need this ?

File: www/front_src/src/Resources/api/endpoint.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const buildParam = ({ name, value }): string => {
+  return `${name}=${JSON.stringify(value)}`;
+};
+
+const buildResourcesEndpoint = ({ state, sort, page, limit }): string => {
+  const params = [
+    {
+      name: 'state',
+      value: [state],
```
Comment (kduret) [inline www/front_src/src/Resources/api/endpoint.ts]: why state is an array ? should we update API contract ?

File: webpack.config.dev.js (hunk 1 above)
Comment (bdauria) [reply webpack.config.dev.js]: When using npm link with centreon-ui, we get two instances of material-ui which leads to errors (same for react-router)

File: www/front_src/src/Resources/api/endpoint.ts (hunk 2 above)
Comment (kduret) [reply www/front_src/src/Resources/api/endpoint.ts]: then, it will be a string. I'll make a PR to update the format

**Comment by bdauria** [comment]
@Thebarda this will be done in a separate PR for all the labels

**Pull request merged**